zuul-core/src/test/java/com/netflix/zuul/netty/server/ServerTest.java

```diff
@@ -68,9 +71,20 @@ public void beforeTest() {
     public void getListeningSockets() throws Exception {
         ServerStatusManager ssm = mock(ServerStatusManager.class);
         Map<NamedSocketAddress, ChannelInitializer<?>> initializers = new HashMap<>();
+        final List<NioSocketChannel> nioChannels = Collections.synchronizedList(new ArrayList<NioSocketChannel>());
```

Comment:
Why is this specific to NIO channels?

Reply:
> Why is this specific to NIO channels?

ServerTest uses **force_nio**  true

```
config.setProperty("zuul.server.netty.socket.force_nio", "true");
```

so the channels should be NioSocketChannel objects

Reply:
Ah, I forgot we updated that in the last change.